specs/core/1_custody-game.md

```diff
@@ -353,13 +372,15 @@ def process_bit_challenge(state: BeaconState,
         domain=get_domain(state, get_current_epoch(state), DOMAIN_CUSTODY_BIT_CHALLENGE),
     )
     # Verify the challenger is not slashed
-    assert challenger.slashed is False
+    assert is_slashable_validator(challenger)
     # Verify the attestation
     assert verify_standalone_attestation(state, convert_to_standalone(state, challenge.attestation))
     # Verify the attestation is eligible for challenging
     responder = state.validator_registry[challenge.responder_index]
-    min_challengeable_epoch = responder.exit_epoch - EPOCHS_PER_CUSTODY_PERIOD * (1 + responder.max_reveal_lateness)
```

Comment:
We can't get rid of the max delay because then the validator would be liable for an unboundedly growing period of data. We also can't set max delay to equal the custody response deadline because that would just be too much data to store (2 MB per epoch * 2**14 epochs = 32 GB); we want a validator to only be responsible for the data in a single period (unless the validator screws up eg. by delaying subkey reveals). So we need a max delay of 1 in the normal case, but if the validator does delay subkey reveals we need to increase the max delay to make sure that when the subkey is revealed the data can still be challenges. This was the rationale for `max_reveal_lateness`.

Reply:
@JustinDrake and I thought about this and couldn't come up with a way to solve this issue without having that variable, despite its ugliness, though what we _can_ do is ameliorate the problem where if you're late once you're stuck with a longer period forever, by having `max_reveal_lateness` be measured in a finer-grained time interval (eg. epochs) and have it slowly decrease over time if a validator shows good behavior (another isomorphic approach is to have a `min_challengeable_epoch` that, if it is more than a full period behind the current epoch, moves forward eg. 2 epochs per epoch.

Reply:
Ah, now I see the rationale for max_reveal_lateness. I like the idea of having it decrease over time.